dotCMS/src/main/java/com/dotcms/security/apps/ParamDescriptor.java

```diff
@@ -14,34 +15,64 @@
 
     private final String hint;
 
-    private final boolean required;
+    private final Boolean required;
 
-    private ParamDescriptor(final String value, final boolean hidden, final Type type, final String label, final String hint, final boolean required) {
+    /**
+     * Private Constructor
+     * @param value
+     * @param hidden
+     * @param type
+     * @param label
+     * @param hint
+     * @param required
+     */
+    private ParamDescriptor(final String value, final Boolean hidden, final Type type, final String label, final String hint, final Boolean required) {
         super(value, hidden, type);
         this.label = label;
         this.hint = hint;
         this.required = required;
     }
 
+    /**
+     * Label getter
+     * @return
+     */
     public String getLabel() {
         return label;
     }
 
+    /**
+     * Hint getter
+     * @return
+     */
     public String getHint() {
         return hint;
     }
 
+    /**
+     * required getter
+     * @return
+     */
     public boolean isRequired() {
+        return UtilMethods.isSet(required) ? required : false ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Ternary operators that can be simplified with || or &&](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5358201)